app/i18n/en/sub.php

```diff
@@ -144,7 +144,7 @@
 			'normal' => 'Show in its category',
 		),
 		'proxy' => 'Set a proxy for fetching this feed',
-		'proxy_help' => 'Select a protocol (e.g: SOCKS5) and enter the proxy address (e.g: <kbd>127.0.0.1:1080</kbd>)',
+		'proxy_help' => 'Select a protocol (e.g: SOCKS5) and enter the proxy address (e.g: <kbd>username:password@127.0.0.1:1080</kbd>)',
```

Comment:
I am afraid that some users might think that providing the username and password is mandatory, so maybe we could indicate both syntax, like:

```suggestion
		'proxy_help' => 'Select a protocol (e.g: SOCKS5) and enter the proxy address (e.g: <kbd>127.0.0.1:1080</kbd> or <kbd>username:password@127.0.0.1:1080</kbd>)',
```

Reply:
Good point, done.